kraft.yaml

```diff
@@ -1,18 +1,20 @@
 specification: '0.5'
 name: helloworld-go
 unikraft:
-  version: '0.5'
+  version: '0.6'
+    kconfig:
```

Comment:
```suggestion
  kconfig:
```

Reply:
My bad, I forgot to add `kraft.yaml` when doing the amending the commit.